clients/da-vinci-client/src/main/java/com/linkedin/davinci/helix/AbstractPartitionStateModel.java

```diff
@@ -248,10 +248,8 @@ protected void removePartitionFromStoreGracefully() {
     // Since this removes the storageEngine from the map not doing an un-subscribe and dropping a partition could
     // lead to NPE and other issues.
     // Adding a topic unsubscribe call for those race conditions as a safeguard before dropping the partition.
-    ingestionBackend.dropStoragePartitionGracefully(
-        storeConfig,
-        partition,
-        getStoreConfig().getPartitionGracefulDropDelaySeconds());
+    ingestionBackend
+        .dropStoragePartitionGracefully(storeConfig, partition, getStoreConfig().getStopConsumptionWaitRetriesNumber());
```

Comment:
This is a little confusing since the function param is actual timeout, not retry:
```
public void dropStoragePartitionGracefully(
      VeniceStoreVersionConfig storeConfig,
      int partition,
      int timeoutInSeconds,
      boolean removeEmptyStorageEngine) 
```

Shall we update the config name to be a timeout instead of retry?
If you think it is reasonable, we should update DaVinci as well and I assume no DaVinci customer is overriding that retry config, so it should be fine to rename the config.
`retry` config doesn't make sense if the `interval` is not configurable or visible.

Reply:
It makes sense. I did a code search, no DVC user configures it. If I change the config name to timeout, we can reuse the graceful drop timeout as its default now, what do you think? Or keep the default as 180 seconds?

Reply:
I think we should use a different default from the graceful drop config, otherwise, it might be confusing.